src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -269,6 +269,17 @@ protected function getConstructor(array &$data, $class, array &$context, \Reflec
     }
 
     /**
+     * @see instantiateComplexObject
+     * @deprecated Since 3.1, will be removed in 4.0. Use instantiateComplexObject instead.
+     */
+    protected function instantiateObject(array &$data, $class, array &$context, \ReflectionClass $reflectionClass, $allowedAttributes)
+    {
+        @trigger_error(sprintf('"%s()" has been deprecated sin Symfony 3.1 and will be removed in version 4.0. Use "%s::instantiateComplexObject()" instead.', __METHOD__, __CLASS__));
```

Comment:
@theofidry phpunit bridge only discovers deprecated errors, not general silenced errors.